recipes/anarci/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "anarci" %}
+{% set version = "2020.04.23" %}
+{% set commit = "19d9de50b2ab7ca80f4d2c45fd113139f4cad311" %}
+{% set sha256 = "e231b2e6b4cccca68f7bf87d129cbdac81e95639ef7725078f81a497b474dfa4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/oxpig/ANARCI/archive/{{ commit }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - "{{ PYTHON }} setup.py install"
```

Comment:
Can you change this to `{{ PYTHON }} -m pip install . -vv --no-deps` and then remove `hmmer` from the host section?

Reply:
I actually tried first with the "pip install" command, but that makes the building process, that generates the HMMs, not work well. I think it probably is an issue upstream.

I will anyway try to see if I can honor some of your suggestions.

Reply:
Ah, good to know. If that fails then please just revert. I'll approve the PR so it can get merged either way.